hypermap/settings/_vcap.py

```diff
@@ -0,0 +1,32 @@
+import dj_database_url
+import json
+import os
+from settings.default import *
+
+vcap_service_config = os.environ.get('VCAP_SERVICES')
```

Comment:
This is the deployment environment of cloud foundry - where we are deploying hypermap.